crates/relayer/src/chain/axon.rs

```diff
@@ -1388,8 +1388,32 @@ impl AxonChain {
                 Ok(e) => return Ok(e),
                 Err(e) => {
                     if e.to_string().contains("reverted: getHeader") {
-                        std::thread::sleep(Duration::from_secs(5));
-                        tracing::info!("getHeader failed, waiting for header sync");
+                        #[cfg(not(test))]
```

Comment:
This may not work as you would expect: https://github.com/synapseweb3/forcerelay/actions/runs/7305634478/job/19909434313?pr=401#step:19:671

(Because only the ibc-test crate would have `cfg(test)`, not the relayer crate.)